tool/ood-gen/lib/event.ml

```diff
@@ -0,0 +1,64 @@
+type location = { lat : float; long : float }
+[@@deriving of_yaml, show { with_path = false }]
+
+ type date_with_optional_time = Date of Ptime.date | DateTime of Ptime.t
+
+type metadata = {
+  title : string;
+  url : string;
+  textual_location : string;
+  location : location option;
+  starts : string;
+  ends : string option;
+}
+[@@deriving of_yaml, show { with_path = false }]
+
+
+type t = {
+  title : string;
+  url : string;
+  slug : string;
+  textual_location : string;
+  location : location option;
+  starts : string;
+  ends : string option;
+  body_md : string;
+  body_html : string;
+}
+[@@deriving
+  stable_record ~version:metadata ~remove:[ slug; body_md; body_html ]]
```

Comment:
add `, show { with_path = false }` to make ppx_show derive the `pp` function that renders values of type `t` to a `string`.

Reply:
Thanks a bunch.